setup.py

```diff
@@ -32,7 +32,7 @@
     python_requires=">=3.6.2, <4",
     install_requires=[
         "chardet",
-        "click>=7.1.1, <8",
+        "click>=8.0.0, <8.1.0",
```

Comment:
We don't need to have an upper-limit here. If someone _is_ still running python 3.6 `click>=8.1.0` simply won't get installed for them. And if 8.1.+ doesn't break anything in our code we don't need to artificially limit it if they are on 3.7+

Reply:
and we can also keep it `>=7.1.1` since we're not depending on anything 8+ specific.

Reply:
oh very cool

Reply:
our unit tests will fail on 7.1.1 by error message asserting but if we're okay with that then sure

Reply:
We could just make the asserts be an `or`, like 
```
assert ("Could not open file: not_a_file" in result.stdout) or (" Invalid value for '--advanced-query': 'not_a_file': No such file or directory" in result.stdout)
```

Reply:
I have no idea why I didn't think of that